src/me/StevenLawson/TotalFreedomMod/Listener/TFM_BlockListener.java

```diff
@@ -111,6 +112,14 @@ else if (playerLocation.distanceSquared(location) > (nukeMonitorRange * nukeMoni
                 }
             }
         }
+        if (player.getGameMode().equals(GameMode.SURVIVAL))
+        {
+            for (ItemStack stack : event.getBlock().getDrops())
+            {
+                event.getBlock();
+                player.getInventory().addItem(stack);
```

Comment:
Additionally, this should probably be in a separate event with `ignoreCancelled = true` and `priority = EventPriority.MONITOR` to only drop items if the event was not cancelled.